doc/source/conf.py

```diff
@@ -26,7 +26,7 @@
 
 # -- Scripts -----------------------------------------------------------------
 # regenerate tables for which residues are selected by keywords
-subprocess.call('./scripts/gen_standard_selections.py')
+# subprocess.call('./scripts/gen_standard_selections.py')
```

Comment:
Maybe comment on how this works or reference a README that says how the docs are build?

If it's not needed then better remove to avoid maintainer confusion.